lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -326,11 +339,33 @@ sub engine_workit {
         my $error = locate_local_assets(\%vars, $assetkeys);
         return $error if $error;
     }
+    my $casedir     = OpenQA::Utils::testcasedir($vars{DISTRI}, $vars{VERSION}, $shared_cache);
+    my $productdir  = OpenQA::Utils::productdir($vars{DISTRI}, $vars{VERSION}, $shared_cache);
```

Comment:
Any reason you don't want to import these functions?

Reply:
I don't change that, just follow the current code.